test/sql/json/scalar/test_json_dot_syntax.test

```diff
@@ -24,17 +24,6 @@ select json('{"foo": null}').foo.bar
 ----
 NULL
 
-# also supports this syntax
-query T
-select json('{"foo": null}')['foo']
-----
-null
-
-query T
-select json('{"foo": null}')['foo']['bar']
-----
-NULL
-
```

Comment:
To clarify, introducing this behavior (allowing indexing strings as well) is totally fine. And would be a good thing imo. Postgres supports that too.

Reply:
Done